taca/backup/backup.py

```diff
@@ -6,6 +6,7 @@
 import subprocess as sp
 import time
 import csv
+import re #added this to allow pattern match
```

Comment:
Already imported here https://github.com/SciLifeLab/TACA/blob/b533241a7a1543f1b82fc1248cfb67ffb54ede32/taca/backup/backup.py#L4

Reply:
I was working on a slightly older version that didn't have it, it was then added when I merged upstream. But I removed my import now.